### PR TITLE
make sure we use a funded address

### LINKED ISSUE #74: commonOwnerOrOperator should give friendlier error when operator address does not yet exist on chain
People have been running into "actor not found" issues because cmds that can be called by owner or operator default to operator, when only the owner address has funds. We should catch these errors by looking up the balance of the caller before we call the method, and throwing a friendlier error message like:

```
Operator address has no balance. Fund the operator, or pass `--from` flag to override
```

### PR DESCRIPTION
**Pull request by lanzafame**
resolves #74.

### CODE REVIEW

File: cmd/utils.go [hunk 1]
```diff
@@ -153,17 +154,32 @@ func commonOwnerOrOperatorSetup(cmd *cobra.Command) (common.Address, *ecdsa.Priv
...
 	// if no flag was passed, we just use the operator address by default
 	from := cmd.Flag("from").Value.String()
-	if from == "" {
-		from = opEvm.String()
-		pk, err = ks.GetPrivate(util.OperatorKey)
-	} else if from == opEvm.String() || from == opFevm.String() {
-		pk, err = ks.GetPrivate(util.OperatorKey)
-	} else if from == owEvm.String() || from == owFevm.String() {
+	switch from {
+	case "", opEvm.String(), opFevm.String():
```
Comment (Schwartz10) [inline cmd/utils.go:158]: What happens here if `from` is `""`? Does the following logic still work?

**Review by Schwartz10 — CHANGES_REQUESTED** [review]
Hey @lanzafame the logic here mostly lgtm. The part I don't love is using `keys.toml` to write funding statuses - i'm feeling more like we shouldn't be touching the keys.toml file for this. I'm just not 100% confident that there are 0 scenarios where something gets overwritten, and we absolutely cannot afford to have a scenario where an owner key gets accidentally overwritten on some type of machine we didn't test on.

Can we use the alerting system for this? Like you should just have an alert "fund the owner key" and then the logic here stays the same. But the caching part uses the journal not keys.toml

File: cmd/utils.go (hunk 1 above)
Comment (lanzafame) [reply cmd/utils.go:158]: Yes, the default was to use the operator if `from` was empty and this maintains that.

File: util/keystore.go [hunk 2]
```diff
@@ -16,9 +17,11 @@ import (
 type KeyType string
 
 const (
-	OwnerKey    KeyType = "owner"
-	OperatorKey KeyType = "operator"
-	RequestKey  KeyType = "request"
+	OwnerKey       KeyType = "owner"
+	OwnerFunded    KeyType = "owner-funded"
```
[the remark this answers is not in the document]
Comment (lanzafame) [reply util/keystore.go:21]: actually, I use `OwnerKeyFunded` to make it clear

**Review by Schwartz10 — APPROVED** [review]

**Pull request merged**